pkg/api/config_test.go

```diff
@@ -32,7 +32,38 @@ func TestValidate(t *testing.T) {
 				},
 			},
 			expectedValid: false,
-			expectedError: "Test should not be called 'images' because it gets confused with '[images]' target",
+			expectedError: "test should not be called 'images' because it gets confused with '[images]' target",
+		},
+		{
+			id: "BuildRoot",
+			config: ReleaseBuildConfiguration{
+				InputConfiguration: InputConfiguration{
+					BuildRootImage: &BuildRootImageConfiguration{
+						ImageStreamTagReference: &ImageStreamTagReference{
+							Cluster:   "test_cluster",
+							Namespace: "test_namespace",
+							Name:      "test_name",
+							Tag:       "test",
+						},
+						GitSourceReference: &GitSourceReference{
+							ContextDir: "/",
+							Dockerfile: "Dockerfile.test",
+						},
+					},
+				},
+			},
+			expectedValid: false,
+			expectedError: "both git_source_image and image_stream_tag cannot be set for the build_root",
+		},
+		{
+			id: "EmptyBuildRoot",
```

Comment:
test names should try to convey what and why -- so:

```go
id: "build root without any content causes an error"
```

This message needs to make sense for someone who has not written the test